### DISCUSSION TITLE
Penggunaan bahasa dalam diskusi

### DISCUSSION THREAD

**Discussion by arfianadam** [reactjs/id.react.dev#361] [Meta]
Halo teman-teman. Bagaimana pendapat Anda tentang penggunaan bahasa di diskusi-diskusi di repo ini? Apa baiknya kita pakai Bahasa Indonesia saja atau tidak mengapa dicampur Bahasa Inggris juga? Menurut pendapat saya, lebih baik pakai Bahasa Indonesia saja.

**Comment by mhaidarhanif** [comment]
The thing is, the main issue and other PRs are mostly already using English by default. But, feel free for the others to input and confirm. Personally, I type faster in English for conversation (except for the translation of course). Mixing is fine, it's still mostly in English anyway.

Namun, issue utama dan di banyak PR sudah kebanyakan pada pakai bahasa Inggris secara default. Tapi, silakan untuk yang lain bisa kasi masukan dan konfirmasi. Secara pribadi saya lebih cepat mengetik dalam bahasa Inggris untuk percakapan (kecuali untuk penerjemahan, tentu saja). Dicampur-campur sedikit masih gak apa-apa, lagipula keliatan kebanyakan sudah bahasa Inggris.

**Reply by mhaidarhanif** [reply]
Read a similar approach with other Indonesian repo, maintained by @zainfathoni: https://github.com/kawalcovid19/wargabantuwarga.com/blob/main/CONTRIBUTING.md#why-are-we-using-english-in-our-issues--prs
